crates/fluvio-cli/src/tableformat/mod.rs

```diff
@@ -62,45 +60,30 @@ impl TableFormatCmd {
     }
 }
 
-//#[derive(Debug, Deserialize, Clone)]
-//pub struct ConnectorConfig {
-//    name: String,
-//    #[serde(rename = "type")]
-//    type_: String,
-//    pub(crate) topic: String,
-//    #[serde(default)]
-//    pub(crate) create_topic: bool,
-//    #[serde(default)]
-//    parameters: BTreeMap<String, String>,
-//    #[serde(default)]
-//    secrets: BTreeMap<String, String>,
-//}
-//
-//impl ConnectorConfig {
-//    pub fn from_file<P: Into<PathBuf>>(path: P) -> Result<ConnectorConfig, CliError> {
-//        let mut file = File::open(path.into())?;
-//        let mut contents = String::new();
-//        file.read_to_string(&mut contents)?;
-//        let connector_config: ConnectorConfig = serde_yaml::from_str(&contents)?;
-//        Ok(connector_config)
-//    }
-//}
-//
-//impl From<ConnectorConfig> for TableFormatSpec {
-//    fn from(config: ConnectorConfig) -> TableFormatSpec {
-//        TableFormatSpec {
-//            name: config.name,
-//            type_: config.type_,
-//            topic: config.topic,
-//            parameters: config.parameters,
-//            secrets: config.secrets,
-//        }
-//    }
-//}
+#[derive(Debug, Deserialize, Clone)]
+pub struct TableFormatConfig {
```

Comment:
Here I am using the exact same format that we use with `kubectl apply`. Is this what we want? or do we want to have something more simpler for the user? @tjtelan

Reply:
We want just the `spec` part of the yaml for `kubectl apply`.

So for a k8 config

```yaml
apiVersion: fluvio.infinyon.com/v1
kind: TableFormat
metadata:
  name: "example-column-ordered"
spec:
  name: "example-column-ordered"
  inputFormat: "JSON"
  columns:
    - keyPath: "key2"
    - keyPath: "key1"
```

We want the user to give a config like:

```yaml
inputFormat: "JSON"
columns:
  - keyPath: "key2"
  - keyPath: "key1"
```

The name is provided as the CLI positional arg

Reply:
got it